Description: XTerm debian configuration.
 Based on Marcelo E. Magallon <mmagallo@debian.org> diff patch.
Author: Rodolfo García Peñas (kix) <kix@kix.es>
Last-Update: 2017-03-11

--- a/WindowMaker/Defaults/WMState.in
+++ b/WindowMaker/Defaults/WMState.in
@@ -10,8 +10,8 @@
         Lock = Yes;
       },
       {
-	Command = xterm;
-	Name = xterm.XTerm;
+	Command = "x-terminal-emulator";
+	Name = "x-terminal-emulator.XTerm";
 	AutoLaunch = No;
 	Forced = No;
 	Position = "0,1";
